spec/models/access_log_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe AccessLog do
+  let(:access_log) { create(:access_log) }
+
+  it 'has valid factories' do
+    expect(build(:access_log)).to be_valid
+  end
+
+  describe '.total_since scope' do
+    subject { described_class.total_since(timestamp) }
+
+    let(:timestamp) { 1.day.ago }
+    let!(:token) { create(:token) }
+
+    let!(:access_logs) do
+      [
+        create(:access_log, token:, timestamp: 1.hour.ago),
+        create(:access_log, token:, timestamp: 2.hours.ago),
+        create(:access_log, token:, timestamp: 3.hours.ago),
+        create(:access_log, token:, timestamp: 2.days.ago)
+      ]
+    end
+
+    it 'fetches only access logs since the given timestamp' do
+      expect(subject).to eq(3)
+    end
```

Comment:
```suggestion
    let!(:recent_access_logs) do
      [
        create(:access_log, token:, timestamp: 1.hour.ago),
        create(:access_log, token:, timestamp: 2.hours.ago),
        create(:access_log, token:, timestamp: 3.hours.ago)
      ]
    end
    let!(:too_old_access_log) { create(:access_log, token:, timestamp: 2.days.ago) }

    it 'fetches only access logs since the given timestamp' do
      is_expected.to contain_exactly(*recent_access_logs)
    end
```

pas sûr de la syntax